chromium_src/components/content_settings/core/browser/content_settings_registry.cc

```diff
@@ -27,6 +29,7 @@ const struct {
     {brave_shields::kBraveShields, ContentSettingsType::BRAVE_SHIELDS},
     {brave_shields::kReferrers, ContentSettingsType::BRAVE_REFERRERS},
     {brave_shields::kCookies, ContentSettingsType::BRAVE_COOKIES},
+    {speedreader::kSpeedreader, ContentSettingsType::BRAVE_SPEEDREADER},
```

Comment:
I don't think you want this here because you don't want to register the default as block. Just register normally like autoplay (but without the delete beforehand). Also the constant is no longer needed I think without this

Reply:
fixed. thanks for the heads up. we should rename that variable to `kBraveShieldsContentSettingstypes`. we should also fix [this comment](https://github.com/brave/brave-core/blob/54aac58d1444960aaf964f6222b7847659ba9961/chromium_src/components/content_settings/core/browser/content_settings_registry.cc#L67) because it adds to the confusion.